.gitignore

```diff
@@ -1,3 +1,6 @@
+.vagrant
```

Comment:
at least pta-intelligent-search-metadata-annotation is referring to the wrong version (1.0-SNAPSHOT)

Reply:
pta-intelligent-search-metadata-extractor and pta-intelligent-search-qa also refer to the wrong version

Reply:
These should be fixed now